components/overflow-group/overflow-group-mixin.js

```diff
@@ -297,8 +296,8 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 				isChomped: false,
 				isHidden: itemHidden,
 				width: Math.ceil(parseFloat(computedStyles.width) || 0)
-					+ parseInt(computedStyles.marginRight) || 0
```

Comment:
We need brackets here to properly calculate the width, otherwise the left margin (which is what is used in RTL) is not added.